templates/rules_debug_element.html.twig

```diff
@@ -0,0 +1,10 @@
+<div class="rules-debug-block">
```

Comment:
If I am reading this right this is a web profiler template. This should not be in rules but rather in the webprofiler module.